### PR TITLE
tentacles_manager 2.0.0 migration

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/28
requires https://github.com/Drakkar-Software/OctoBot/pull/1073

### CODE REVIEW

File: Backtesting/importers/exchanges/__init__.py [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-#  Lesser General Public License for more details.
-#
-#  You should have received a copy of the GNU Lesser General Public
-#  License along with this library.
+from octobot_tentacles_manager.api.inspector import check_tentacle_version
+from octobot_commons.logging.logging_util import get_logger
+
+if check_tentacle_version('1.2.0', 'generic_exchange_importer', 'OctoBot-Default-Tentacles'):
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Backtesting/importers/exchanges/__init__.py]: It is, these init files are generated files, we need to figure out a way to handle devs on this repo, I'll clean it up

File: Backtesting/converters/exchanges/__init__.py [hunk 2]
```diff
@@ -1,17 +1,8 @@
-#  Drakkar-Software OctoBot-Tentacles
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Backtesting/converters/exchanges/__init__.py]: generated file, see next comment

File: Interfaces/web_interface/controllers/commands.py [hunk 3]
```diff
@@ -41,3 +41,41 @@ def commands(cmd=None):
                            cmd=cmd,
                            metrics_enabled=get_metrics_enabled(),
                            disclaimer=DISCLAIMER)
+
+
+@server_instance.route("/test")
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Interfaces/web_interface/controllers/commands.py]: ahaha, I completely forgot about this, i'll remove it

File: Interfaces/web_interface/controllers/tentacles.py [hunk 4]
```diff
@@ -37,18 +37,17 @@ def _handle_package_operation(update_type):
             path_or_url, action = next(iter(request_data.items()))
             path_or_url = path_or_url.strip()
             if action == "description":
-                package_description = get_tentacles_package_description(path_or_url)
+                package_description = {"name": "#TODO"}
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Interfaces/web_interface/controllers/tentacles.py]: not yet, it was used before to display the name of the package to be installed, now we have to figure out it it's still relevant or not

File: Backtesting/importers/exchanges/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Herklos) [reply Backtesting/importers/exchanges/__init__.py]: I think the best solution is to generate zip from a clean OctoBot-Tentacles repository. I don't think this is a good idea to add generated files / codes in the repo.

**Pull request merged**